pkg/metrics/metrics.go

```diff
@@ -214,6 +217,13 @@ var (
 
 	// L7 statistics
 
+	// ProxyRedirects is the number of redirects labelled by protocol
+	ProxyRedirects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "proxy_redirects_total",
```

Comment:
A gauge should never have the suffix `_total`.

Reply:
I think that @eloycoto should actually change it from gauge to counter, it will simplify things a bit.